dependencies.go

```diff
@@ -0,0 +1,149 @@
+package opslevel
+
+import "fmt"
+
+type ServiceDependency struct {
+	Id     ID        `graphql:"id"`
+	Parent ServiceId `graphql:"sourceService"`
+	Child  ServiceId `graphql:"destinationService"`
+	Notes  string    `graphql:"notes"`
+}
+
+type ServiceDependenciesEdge struct {
+	Id     ID         `graphql:"id"`
+	Locked bool       `graphql:"locked"`
+	Node   *ServiceId `graphql:"node"`
+	Notes  string     `graphql:"notes"`
+}
+
+type ServiceDependentsEdge struct {
+	Id     ID         `graphql:"id"`
+	Locked bool       `graphql:"locked"`
+	Node   *ServiceId `graphql:"node"`
+	Notes  string     `graphql:"notes"`
+}
+
+type ServiceDependenciesConnection struct {
+	Edges    []ServiceDependenciesEdge `graphql:"edges"`
+	PageInfo PageInfo
+}
+
+type ServiceDependentsConnection struct {
+	Edges    []ServiceDependentsEdge `graphql:"edges"`
+	PageInfo PageInfo
+}
+
+type ServiceDependencyKey struct {
+	Parent IdentifierInput `json:"sourceIdentifier"`
+	Child  IdentifierInput `json:"destinationIdentifier"`
+}
+
+type ServiceDependencyCreateInput struct {
```

Comment:
Is there not going to be an update function?

Reply:
Theres a discussion about this with Ownership team.  If they can fit in making one we can implement it.